test/development/acceptance/error-recovery.test.ts

```diff
@@ -369,28 +370,28 @@ describe('pages/ error recovery', () => {
          "stack": [],
        }
       `)
-    } else if (process.env.NEXT_RSPACK) {
+    } else if (isRspack) {
       await expect({ browser, next }).toDisplayRedbox(`
        {
          "description": "  × Module build failed:",
          "environmentLabel": null,
          "label": "Build Error",
          "source": "./index.js
          × Module build failed:
-         ├─▶   ×
-         │     │   x Expected '{', got 'return'
-         │     │    ,-[<FIXME-project-root>/index.js:5:1]
-         │     │  2 |
-         │     │  3 | class ClassDefault extends React.Component {
-         │     │  4 |   render()
-         │     │  5 |     return <h1>Default Export</h1>;
-         │     │    :     ^^^^^^
-         │     │  6 |   }
-         │     │  7 | }
-         │     │    \`----
-         │     │
-         │
-         ╰─▶ Syntax Error
+         ╰─▶   × Error:   x Expected '{', got 'return'
+               │    ,-[5:1]
+               │  2 |
```

Comment:
same with this one, not sure if lack of FIXME must be flagged as failing test case

Reply:
I think this is fine?